Jenkinsfile

```diff
@@ -58,6 +58,7 @@ pipeline {
           setEnvVar('GO_MOD_CHANGES', isGitRegionMatch(patterns: [ '^go.mod' ], shouldMatchAll: false).toString())
           setEnvVar('PACKAGING_CHANGES', isGitRegionMatch(patterns: [ '^dev-tools/packaging/.*' ], shouldMatchAll: false).toString())
           setEnvVar('GO_VERSION', readFile(".go-version").trim())
+          setEnvVar('GODEBUG', "x509ignoreCN=0")
```

Comment:
with the updates to certificate creation, do we still need to set this flag? Using "GODEBUG" here might break in the future right? Will we have to update tests again then?

Reply:
It is not needed. However, I kept these environment variables to make sure we are running go with the same env variables as our users.

Reply:
👍